pkg/vcs/openbsd.go

```diff
@@ -7,8 +7,8 @@ type openbsd struct {
 	*git
 }
 
-func newOpenBSD(vm, dir string) *openbsd {
+func newOpenBSD(dir string, user bool) *openbsd {
```

Comment:
`bool` seem like a fairly suboptimal type for something named `user`. Can this be an enum?

Reply:
Done